class-tgm-plugin-activation.php

```diff
@@ -457,11 +450,11 @@ public function add_plugin_action_link_filters() {
 		 * @since 2.5.0
 		 *
 		 * @param array $actions Action links.
-		 *
```

Comment:
I've since chatted to @DrewAPicture and he's confirmed that the WP standard is to group `@param` and `@return` together (as oer the WP CS handbook), so that trumps our preference and PSR-5.